compilationunit.lua

```diff
@@ -62,7 +62,8 @@ function premake.extensions.compilationunit.customBakeFiles(base, prj)
 		end
 
 		-- add the compilation units for premake
-		for i = 1, cu.numcompilationunits do
+                local count = math.min(#cu.compilationunits[cfg], cu.numcompilationunits)
```

Comment:
sorry for being picky, but tabs please :)

Reply:
Consistency is important in code, so pickiness is welcome :)